src/routes/relay/entities/schema/sponsored-call.schema.helper.ts

```diff
@@ -353,17 +371,12 @@ export const isValidCreateProxyWithNonceCall = (
 export const getOwnersFromCreateProxyWithNonce = (
   encodedData: string,
 ): string[] => {
-  const SETUP_FRAGMENT =
-    'function setup(address[] calldata _owners, uint256 _threshold, address to, bytes calldata data, address fallbackHandler, address paymentToken, uint256 payment, address payable paymentReceiver) external';
-
   const { initializer } = decodeCreateProxyWithNonce(encodedData);
 
-  const setupInterface = new ethers.Interface([SETUP_FRAGMENT]);
+  const safeInterface = Gnosis_safe__factory.createInterface();
+  const setupFragment = safeInterface.getFunction('setup');
 
-  const [owners] = setupInterface.decodeFunctionData(
-    SETUP_FRAGMENT,
-    initializer,
-  );
+  const [owners] = safeInterface.decodeFunctionData(setupFragment, initializer);
```

Comment:
We do not need the fragment here IMO

```suggestion
  const [owners] = safeInterface.decodeFunctionData("setup", initializer);
```

Reply:
Whilst I agree that it's cleaner, we lose type safety by doing this:

![image](https://user-images.githubusercontent.com/20442784/234532966-1604f640-0bb6-4556-ad86-00a7223ccf47.png)